project/Dependencies.scala

```diff
@@ -7,6 +7,10 @@ object Dependencies {
     "org.specs2" %% "specs2-scalacheck" % specs2Version % Test
   )
 
+  val mockito = Seq(
+    "org.specs2" %% "specs2-mock" % specs2Version % Test
```

Comment:
since this is a specs2 module, perhaps it would be better to put this together with the other specs2 dependencies